orc8r/cloud/go/mproto/mproto_test.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mproto_test
+
+import (
+	b64 "encoding/base64"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"magma/orc8r/cloud/go/mproto"
+	mocks "magma/orc8r/cloud/go/mproto/mocks"
```

Comment:
Hm, you know why this is self-aliased?

Reply:
I think it's because the original `mocks.proto` (now `test.proto`) actually belongs to the package `protos`, but in this file there's another proto file imported that also belongs to `protos`, and therefore this one had to be renamed to `mocks` to avoid duplication

Reply:
The latest commit should resolve this